### PR TITLE
zsh-vi-mode 0.8.2

### PR DESCRIPTION
**Pull request by imjma**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/zsh-vi-mode.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  desc "Better and friendly vi(vim) mode plugin for ZSH"
+  homepage "https://github.com/jeffreytse/zsh-vi-mode"
+  url "https://github.com/jeffreytse/zsh-vi-mode/archive/refs/tags/v0.8.2.tar.gz"
+  sha256 "888d68b451fc3221c2ee1648dbfbff38e524048b2c78620a244650fc76946c28"
+  license "MIT"
+
+  bottle :unneeded
+
+  def install
+    pkgshare.install "zsh-vi-mode.zsh"
```
Comment (carlocab) [inline Formula/zsh-vi-mode.rb:11]: Why install this into `pkgshare`, and not the `prefix`? We should probably install `zsh-vi-mode.plugin.zsh` too.

File: Formula/zsh-vi-mode.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  end
+
+  def caveats
+    <<~EOS
+      To activate the zsh vi mode, add the following at the end of your .zshrc:
+        source #{HOMEBREW_PREFIX}/share/zsh-vi-mode/zsh-vi-mode.zsh
```
Comment (carlocab) [inline Formula/zsh-vi-mode.rb:17]:
```suggestion
        source #{opt_pkgshare}/zsh-vi-mode.zsh
```

File: Formula/zsh-vi-mode.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      You will also need to force reload of your .zshrc:
+        source ~/.zshrc
```
Comment (carlocab) [inline Formula/zsh-vi-mode.rb:19]: This isn't great advice, as `.zshrc`s can often be not idempotent. Preferably, this should just suggest restarting the terminal.

File: Formula/zsh-vi-mode.rb (hunk 1 above)
Comment (imjma) [reply Formula/zsh-vi-mode.rb:11]: I followed install in zsh-autosuggestions.rb and zsh-history-substring-search.rb
yes, adding zsh-vi-mode.plugin.zsh to the install

File: Formula/zsh-vi-mode.rb (hunk 2 above)
Comment (imjma) [reply Formula/zsh-vi-mode.rb:17]: done

**Review by carlocab — APPROVED** [review]
Thanks, @imjma.

**Pull request merged**